services/core/src/sourcify-chains.ts

```diff
@@ -148,5 +148,22 @@ export default {
         "supported": true,
         "monitored": true,
         "graphQLFetchAddress": "https://rinkeby-indexer.arbitrum.io/graphql"
+    "40": {
```

Comment:
I'm guessing there were some conflicts when rebasing, so this is now malformed (a `},` missing).

Reply:
Also, not really necessary, but the entries in this file are numerically sorted by key, so you can change the position too if you want to.

Reply:
Yeah, that must've happened in the rebase, added the line back in.